resources/projects/s2i-dotnetcore-ex/app/app.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
I think it makes sense to keep this as is (e.g net6.0) as there is already a new test case for net8.0 added above.